lsstsal/scripts/code/templates/SALDDS_pybind11.cpp.template

```diff
@@ -63,12 +63,15 @@ PYBIND11_MODULE(SALPY_SALData,m){
 // INSERT_SAL_PYTHON_DATATYPES
     py::class_<SALData_ackcmdC>(m,"SALData_ackcmdC" )    
         .def(py::init<>())
-        .def(py::init([](int a, int b, const std::string &name))
```

Comment:
I suggest retaining this (especially if it was part of the original code) and adding defaults and named arguments, e.g. change it to:
```
.def(py::init<int, int, const std::string &>(), “ack”_a, “error”_a=1, “result”_a=“”)
```
which requires the following line somewhere in the file (preferably near the top, just after the includes):
```
using namespace pybind11::literals;
```
Then you won't need the `AckCmd` class (which I see you are removing in the next commit).

Reply:
This was not in the original code, this merge is to essentially remove the commits I made when breaking develop

Reply:
OK. But if the constructor exists then you might consider wrapping it with this minimal form:
```
.def(py::init<int, int, const std::string &>())
```
Adding named arguments and defaults is probably too much for this branch. It should work just fine, but finding a place to put `using namespace pybind11::literals` may be a hassle (if and when you do it I'd look for `namespace py = pybind11` and putting it right after that -- they definitely belong together).